src/quicr_client_raw_session.h

```diff
@@ -275,7 +276,10 @@ class ClientRawSession
   bool has_shared_transport{ false };
   std::atomic_bool stopping{ false };
   qtransport::StreamId transport_dgram_stream_id{ 0 };
-  qtransport::TransportContextId transport_context_id;
+  // XXX(richbarn): I added an initializer for this variable for this member
+  // that's of the same form as the ones above.  But it's not clear to me that
+  // this is actually the right answer.
+  qtransport::TransportContextId transport_context_id{ 0 };
```

Comment:
It may be, but then some added checking should probably go into the constructor or the `connect` method:
```
if context_id == 0
  throw We haven't started transport! Panic!!!

Reply:
Looking at the code, it seems that this field is [not actually initialized until `connect()`](https://github.com/Quicr/libquicr/blob/main/src/quicr_client_raw_session.cpp#L92C45-L92C45).  So I would suggest changing its type to `std::optional<qtransport::TransportContextId>`.  That will make the initialization error go away, and will force other methods to check that it is present (and thus the transport is connected) before using it.  (As opposed to just [blindly assuming it is initialized](https://github.com/Quicr/libquicr/blob/main/src/quicr_client_raw_session.cpp#L260).)  It's the safer, clearer way of doing your `if context_id == 0` trick.